### ISSUE TITLE
Config Manager - List Settings Lost When Clicking Away and Then Back

### ISSUE THREAD

**Issue by dsmithqld** [DMS-Aus/Roam#269]
If I have a field configured with a List and I click on another field and then back on that field, the config gets wiped.

**Comment by NathanW2** [comment]
Hmm I don't see that here at all.

**Comment by bzeeb** [comment]
I've seen this happen before, can't remember what issue/fix was, will give it a shot.

**Comment by bzeeb** [comment]
Couldn't reproduce in newest builds, let us know if you still see it happening after 2.4 release.

**Comment by bzeeb** [comment]
Been testing and using 2.4 config manager for new projects all day, have not seen this issue in new version so closing for now. Re-open if you can find steps to reproduce.

**Comment by dsmithqld** [comment]
This can be reproduced by saving, exiting config manager and then re-opening it and switching between form field configs.  The layer list values will disappear.  It must be a config loading/binding issue.  Please re-open.

**Comment by NathanW2** [comment]
I can confirm this now too.  That is a bit annoying.

**Comment by rjwillson** [comment]
yep, can also confirm. It can be beaten into submission by patiently setting it up a few times. At least I was able to get two fields bound to different lists on the same form but I would be afraid to add additional fields. I have been testing 2.4 in the field for the last few days (more coming this week) and I just want to say that the features and functionality are outstanding. My wife gets tired of hearing me go on and on about how great Roam and QGIS are so I had to vent some praise here as well.
